packages/@okta/vuepress-site/docs/reference/beta-only/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: APIs only documented on the new beta reference
+---
+
+# APIs documented only on the new beta reference
+
+We've got a new API reference in the works! With a fresh look and feel, our new API content features a more logical navigation and a wider variety of code examples. [Try the beta now](https://developer.okta.com/docs/api/) and help us improve the site by [providing feedback](https://forms.gle/Y9XmNNTF2rPQwive7).
+
+In some cases, APIs have only been documented on the new beta reference site, and not on the old API reference that you are currently looking at. This page provides a list of those, so they can be found more easily.
```

Comment:
```suggestion
In some cases, APIs have only been documented on the new beta reference site, and not on the old API reference that you are currently looking at. This page provides a list of those, so that you can easily find them.
```